examples/profile-client/src/main/java/com/palantir/example/profile/ProfileStore.java

```diff
@@ -49,22 +50,18 @@ public ProfileStore(TransactionManager txnMgr, Transaction t) {
         this.t = t;
     }
 
-    private long getNewId() {
-        long result = -1;
-        while (result < 0) {
-            result = new Random().nextLong();
-        }
-        return result;
+    private UUID getNewId() {
+        return new UUID(new Random().nextLong(), new Random().nextLong());
```

Comment:
I'm happy to use UUID.randomUUID(), but heard that SecureRandom (used by the random uuid generation) has/causes entropy issues.